playbooks/tests/tasks/ceph.yml

```diff
@@ -13,3 +13,64 @@
   tasks:
   - name: cinder uses rbd_hybrid or rbd_ssd as backend
     shell: grep -P "^enabled_backends.+rbd_(hybrid|ssd)" /etc/cinder/cinder.conf
+
+- name: restart ceph osd service,and check ceph status
+  hosts: ceph_osds_ssd[0]:ceph_osds_hybrid[0] 
+  tasks:
+  - name: restart ceph osd service(rhel) 
+    service: 
+      name: ceph-osd.target
+      state: restarted
+    when: ursula_os == 'rhel'
+
+  - name: restart ceph osd service(ubuntu)
+    service:
+      name: ceph-osd-all
+      state: restarted
+    when: ursula_os == 'ubuntu'
+
+  - name: check ceph cluster health
+    shell: ceph health 
+    register: result
+    until: result.stdout == 'HEALTH_OK'
+    retries: 3
+    delay: 30
+
+- name: restart ceph mon service , and check ceph status
+  hosts: ceph_monitors[0]
+  tasks:
+  - name: restart ceph mon service(rhel)
+    service:
+      name: ceph-mon.target
+      state: restarted
+    when: ursula_os == 'rhel'
+
+  - name: restart ceph mon service(ubuntu)
+    service: 
+      name: ceph-mon-all
+      state: restarted
+    when: ursula_os == 'ubuntu'
+
+  - name: check ceph cluster health
+    shell: ceph health 
+    register: result
+    until: result.stdout == 'HEALTH_OK'
+    retries: 3
+    delay: 30
+
+- name: reboot ceph_osds node, and check ceph status
+  hosts: ceph_osds[0]:ceph_monitors[0]
+  serial: 1
+  tasks:
+    - name: reboot ceph_osds|ceph_monitor nodes
+      shell: (sleep 5 ; shutdown -r now )&
```

Comment:
there is no `-t ` option for `shutdown` now, the min option for shutdown is minutes